Bottom: 541f42373a79783d9770fa08a56903d8f30a9db9
Top:    333451af10a01bda1801e215e75b6c9063c16ad1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2013-06-05 12:07:58 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 357e645..f9ff3f6 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -183,7 +183,7 @@ DrawStrippedText (hidGC gc, ElementType *Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
